src/content/docs/style-guide/writer-workflow/github-intro.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Get around GitHub
+topics:
+  - Tech writer style guide
+  - How to use GitHub 
+---
+As tech doc writers (TW) we [edit docs](/docs/style-guide/writer-workflow/tech-writer-workflow/), do [peer edits](/docs/style-guide/writer-workflow/peer-editor-workflow/), or use the [Docs Team GitHub board](https://github.com/newrelic/docs-website) to track the status of issues and pull requests (PR).  
+
+## Who is who in an issue/PR? [#who-is-who]
+GitHub keeps track of all activity concerning an issue or PR, including, of course, the people involved. When a new issue or PR is filed, check on the filer’s username and see if they're listed as a member of our organization. If you're not sure, treat someone as external until you know otherwise. 
+
+People in an issue/PR include:
+* Assignee: The person taking responsibility for a PR or issue. This will usually be used by the Hero or Sidekick to assign non-TW PRs and issues to themselves. It can also be used to take a TW’s PR or issue over from them.
+* Reviewer: The person who reviews or peer edits the code/document and approves the changes. Not necessarily the person responsible for that area or responsible for merging the commit. You can pre-assign up to 100 reviewers to a given issue.
+
+## Track issues in the board [#github-board] 
+The docs board has the following columns:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "300px" }}>
+        Column
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        **Needs triage**
+      </td>
+      <td>
+        The Hero or Sidekick review issues and PRs in this column, then drag them to the appropriate column.
+
+        If the issue/PR is labeled `eng`, go ahead and click its ellipses icon to archive it. For other issues and PRs, add these labels as applicable:
+        * `content`
+        * `internal`, when issued by a New Relic employee
+        * `external`, when issued by someone outside New Relic, or you're not sure they are an employee
+        * `Jira’d`, if you create a Jira from the issue  
+
+        If you create a Jira, don't close the original issue/PR, but feel free to remove it from the board until you work on it again.
+        
+        If you open an PR based on an external issue, label that PR as internal since you're doing the work.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        **Hero: to do**
+      </td>
+      <td>
+        PRs that the Hero needs to review, publish, and  follow up with SMEs as needed.  
+        
+        **Assign yourself as Assignee**.
```

Comment:
Maybe it’s worth specifying that it’s the hero who will turn into the Assignee: `Hero: assign yourself as Assignee.`

Reply:
Good point!